src/stateReconciler/autoMergeLevel2.js

```diff
@@ -1,6 +1,6 @@
 
 const merge = (state, payload) => {
-  return state.mergeDeepWith((oldVal, newVal) => newVal || oldVal, payload);
```

Comment:
@lukasikora what is the point of removing?
I add it cause: if you added something to the reducer, but it is missing in the persisted data - rehydration will overwrite it to null.